plugins/guisupport/guisupport.cpp

```diff
@@ -82,6 +84,14 @@ Q_DECLARE_METATYPE(QPixelFormat::ByteOrder)
 Q_DECLARE_METATYPE(QPixelFormat::ColorModel)
 Q_DECLARE_METATYPE(QPixelFormat::TypeInterpretation)
 Q_DECLARE_METATYPE(QPixelFormat::YUVLayout)
+Q_DECLARE_METATYPE(Qt::MouseButton)
+Q_DECLARE_METATYPE(QFlags<Qt::MouseEventFlag>)
```

Comment:
Elements in the Qt namespace typically have metatypes already via a Q_ENUM/Q_FLAG special-case. Sure we need those here?

Reply:
This doesn't compile, either, and even with those lines the MouseEventFlags are not correctly shown in the UI :thinking: